tests/Xendit/EWalletsTest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/**
+ * EWalletsTest.php
+ * php version 7.2.0
+ *
+ * @category Test
+ * @package  Xendit
+ * @author   Ellen <ellen@xendit.co>
+ * @license  https://opensource.org/licenses/MIT MIT License
+ * @link     https://api.xendit.co
+ */
+
+namespace Xendit;
+
+use Xendit\EWallets;
+use Xendit\TestCase;
+
+/**
+ * Class EWalletsTest
+ *
+ * @category Class
+ * @package  Xendit
+ * @author   Ellen <ellen@xendit.co>
+ * @license  https://opensource.org/licenses/MIT MIT License
+ * @link     https://api.xendit.co
+ */
+class EWalletsTest extends TestCase
+{
+    const TEST_RESOURCE_ID = "123";
+    const TEST_RESOURCE_TYPE = "DANA";
+
+    /**
+     * Create EWallet test
+     * Should pass
+     *
+     * @return void
+     * @throws Exceptions\ApiExceptions
+     */
+    public function testIsCreatable()
+    {
+        $params = [
+            'external_id' => 'demo_' . time(),
+            'amount' => 32000,
+            'phone' => '081298498259',
+            'expiration_date' => '2020-02-20T00:00:00.000Z',
+            'callback_url' => 'https://my-shop.com/callbacks',
+            'redirect_url' => 'https://my-shop.com/home',
+            'ewallet_type' => 'DANA'
+        ];
+
+        $this->stubRequest(
+            'POST',
+            '/ewallets',
+            $params,
+            [],
+            $params
+        );
+
+        $resource = EWallets::create($params);
```

Comment:
For unit testing, can you write the assertion variable to result and expected result?
Because right now if I read the `assertEquals` directly without knowing the above line, I don't know if `$resource` is the result or expected result.

My suggestion is you can do it like this
```
$params = [
...
]

$expectedResult = $params

...
...

$result = EWallets::create($params)

$this->assertEquals($result['external_id'], $expectedResult['external_id'])
```

If you do it that way, it is easier for people who don't know PHPUnit. They know what is inside the first and second argument of the assertEquals function

Reply:
Good point, thanks